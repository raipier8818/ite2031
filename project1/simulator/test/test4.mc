8454150
21626880
12779527
25165824
4194307
22085632
4
0
