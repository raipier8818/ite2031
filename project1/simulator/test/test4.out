memory[0]=8454150
memory[1]=21626880
memory[2]=12779527
memory[3]=25165824
memory[4]=4194307
memory[5]=22085632
memory[6]=4
memory[7]=0

@@@
state:
	pc 0
	memory:
		mem[ 0 ] 8454150
		mem[ 1 ] 21626880
		mem[ 2 ] 12779527
		mem[ 3 ] 25165824
		mem[ 4 ] 4194307
		mem[ 5 ] 22085632
		mem[ 6 ] 4
		mem[ 7 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 1
	memory:
		mem[ 0 ] 8454150
		mem[ 1 ] 21626880
		mem[ 2 ] 12779527
		mem[ 3 ] 25165824
		mem[ 4 ] 4194307
		mem[ 5 ] 22085632
		mem[ 6 ] 4
		mem[ 7 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 4
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 4
	memory:
		mem[ 0 ] 8454150
		mem[ 1 ] 21626880
		mem[ 2 ] 12779527
		mem[ 3 ] 25165824
		mem[ 4 ] 4194307
		mem[ 5 ] 22085632
		mem[ 6 ] 4
		mem[ 7 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 4
		reg[ 2 ] 2
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 5
	memory:
		mem[ 0 ] 8454150
		mem[ 1 ] 21626880
		mem[ 2 ] 12779527
		mem[ 3 ] 25165824
		mem[ 4 ] 4194307
		mem[ 5 ] 22085632
		mem[ 6 ] 4
		mem[ 7 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 4
		reg[ 2 ] 2
		reg[ 3 ] -1
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 2
	memory:
		mem[ 0 ] 8454150
		mem[ 1 ] 21626880
		mem[ 2 ] 12779527
		mem[ 3 ] 25165824
		mem[ 4 ] 4194307
		mem[ 5 ] 22085632
		mem[ 6 ] 4
		mem[ 7 ] 0
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 6
		reg[ 2 ] 2
		reg[ 3 ] -1
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 3
	memory:
		mem[ 0 ] 8454150
		mem[ 1 ] 21626880
		mem[ 2 ] 12779527
		mem[ 3 ] 25165824
		mem[ 4 ] 4194307
		mem[ 5 ] 22085632
		mem[ 6 ] 4
		mem[ 7 ] -1
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 6
		reg[ 2 ] 2
		reg[ 3 ] -1
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state
machine halted
total of 6 instructions executed
final state of machine:

@@@
state:
	pc 4
	memory:
		mem[ 0 ] 8454150
		mem[ 1 ] 21626880
		mem[ 2 ] 12779527
		mem[ 3 ] 25165824
		mem[ 4 ] 4194307
		mem[ 5 ] 22085632
		mem[ 6 ] 4
		mem[ 7 ] -1
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 6
		reg[ 2 ] 2
		reg[ 3 ] -1
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state
